warehouse/models/mart/ntd_annual_reporting/dim_2022_agency_information.sql

```diff
@@ -3,9 +3,25 @@ WITH staging_agency_information AS (
     FROM {{ ref('stg_ntd__2022_agency_information') }}
 ),
 
+current_dim_organizations AS (
+    SELECT
+        ntd_id,
+        caltrans_district
```

Comment:
I believe this is still extracting the Caltrans district from dim_organizations which extracts it form a deprecated column in Airtable. We should probably remove Caltrans District from the dim_organizations table.

Reply:
See [this Caltrans District enrichment of Dim Organizations Demo metabase question](https://dashboards.calitp.org/question/3433-caltrans-district-enrichment-of-dim-organizations-demo)